app-text/xapers/xapers-0.8.2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6})
```

Comment:
does not it work with py3.7 yet?

Reply:
I have not tested with py3.7, so I do not know. However, enabling py3.7 adds a whole heap of fatal errors to repoman. Should I add it despite these issues?

Reply:
No, that usually means that some of the required dependencies arent updated to python3_7 yet. However, you should read the repoman errors, they are quite clear of causes ;)

Another thing. You should check if *app-text/poppler* needs a slot operator, `:=`, especially because it's required in `DEPEND`.

Reply:
I checked, and the errors are caused by dependencies (in particular, xapian-bindings, urwid and bibtexparser) not having python3_7 yet, and no other errors are listed by repoman. I just didn't know whether the correct process is to add python3_7 to this package despite not testing and dependencies not being ready, or whether the correct thing to do is to wait for dependencies.

As for the poppler advice, thanks. I didn't actually know, so I went digging and it turns out the module just calls the command line tool pdftotext, so I've amended the commit to add [utils] to the poppler to ensure that this utility is available. I also scanned the source to find other executables it might call, and I found both xclip and xdg-open hard coded, so I added xclip and xdg-utils to RDEPEND.